e3db/types/note_options.py

```diff
@@ -0,0 +1,81 @@
+from datetime import date
+
+# Look for a possible alternative to a NoteOptions class. Not a huge priority at the moment, 
+# but this may not be what is wanted. 
+class NoteOptions():
+    
+    def __init__(self, client_id: str, max_views: int, note_name: str, expiration: date, # Check `date` type for expiration. 
```

Comment:
Consider making `client_id` more explicit / [intention revealing](https://wiki.c2.com/?IntentionRevealingNames), e.g. note_writer_client_id or `note_recipient_client_id` (looking at this code I can’t tell which one of those this is)

Reply:
I think in all contexts we use and non explicit `client_id` to mean the writer clientID but I see no problem calling it writer_client_id. With the understanding that it might also be the default UUID for anonymous notes.